src/models/curriculum/unit.ts

```diff
@@ -40,7 +40,8 @@ export const UnitModel = types
     investigations: types.array(InvestigationModel),
     supports: types.array(SupportModel),
     defaultStamps: types.array(StampModel),
-    settings: types.maybe(SettingsMstType)
+    settings: types.maybe(SettingsMstType),
+    navTabs: types.optional(NavTabsAppConfigModel, () => NavTabsAppConfigModel.create())
```

Comment:
```suggestion
    navTabs: types.maybe(NavTabsAppConfigModel)
```
Creating a default model at the unit level will just confuse the issue when it comes time to find the appropriate model to use.